website/search/views.py

```diff
@@ -196,9 +196,10 @@ def search_share():
     results = {}
 
     is_count = request.args.get('count') is not None
+    raw = request.args.get('raw') is not None
 
     if request.method == 'POST':
```

Comment:
having `query` defined in the `if request.method == ...` and then having a single if block for `is_count` maybe a bit more clear.

```python
if post:
    query = ...
else:
    query = ...

if count:
    results = search(query, raw)
else:
    results = search(query, raw)
```